### PR TITLE
chore: Replace embedURL element with Icon/spinner components

### PR DESCRIPTION
**Pull request by GoOz**
https://github.com/cozy/cozy-ui/pull/904 is removing the old spinner placeholders and such from UI and Drive & Photos were still using them.
So I removed those placeholders and use `<Spinner />` or `<Icon />` when relevant instead.

### CODE REVIEW

File: src/photos/ducks/upload/styles.styl [hunk 1]
```diff
@@ -130,30 +130,6 @@ progress.upload-queue-progress::-moz-progress-bar
...
         line-height    2rem
 
-    .item-loading
-        @extend $icon-16
-        @extend $icon-spinner-blue
-
-    .item-canceled
-        @extend $icon-16
-        @extend $icon
-        background-image embedurl('./assets/icons/icon-cross.svg')
```
Comment (Crash--) [inline src/photos/ducks/upload/styles.styl]: these SVG are still used somewhere else or can we remove them?

**Comment by GoOz** [comment]
[(𝐒) [cozy-ui] Replace remaining embedurl (tous sauf le formulaire)](https://trello.com/c/zD7HtKuc/1284-%F0%9D%90%92-cozy-ui-replace-remaining-embedurl-tous-sauf-le-formulaire)

File: src/photos/ducks/upload/styles.styl (hunk 1 above)
Comment (GoOz) [reply src/photos/ducks/upload/styles.styl]: Right! Apparently not used anymore. Deleted now! :)

**Comment by GoOz** [comment]
What should I do with the visual regression the cozy-bot posted?

**Comment by GoOz** [comment]
So what's the next step here ? Visual regression is wanted and ok but the CI seems to break on Photos without any particular error…  :|

**Comment by y-lohse** [comment]
Can you rebase the PR and see if it's any better?

**Pull request merged**